engine/apps/email/tests/test_notify_user.py

```diff
@@ -0,0 +1,118 @@
+import socket
+from unittest.mock import patch
+
+import pytest
+from django.core import mail
+from django.core.mail.backends.locmem import EmailBackend
+
+from apps.base.models import UserNotificationPolicy, UserNotificationPolicyLogRecord
+from apps.email.tasks import notify_user_async
+from apps.user_management.subscription_strategy.free_public_beta_subscription_strategy import (
+    FreePublicBetaSubscriptionStrategy,
+)
+
+
+@pytest.mark.django_db
+def test_notify_user(
+    settings,
+    make_organization,
+    make_user_for_organization,
+    make_token_for_organization,
+    make_alert_receive_channel,
+    make_alert_group,
+    make_alert,
+    make_user_notification_policy,
+):
+    settings.EMAIL_BACKEND = "django.core.mail.backends.locmem.EmailBackend"
+
+    organization = make_organization()
+    user = make_user_for_organization(organization)
+
+    alert_receive_channel = make_alert_receive_channel(organization)
+    alert_group = make_alert_group(alert_receive_channel)
+
+    make_alert(alert_group=alert_group, raw_request_data=alert_receive_channel.config.example_payload)
+
+    notification_policy = make_user_notification_policy(
+        user,
+        UserNotificationPolicy.Step.NOTIFY,
+        notify_by=587,
+        important=False,
+    )
+
+    notify_user_async(user.pk, alert_group.pk, notification_policy.pk)
```

Comment:
where are smtp settings defined for tests?

Reply:
Since `django.core.mail.backends.locmem.EmailBackend` is used as the django email backend, SMTP credentials are not needed and will be disregarded in case it's provided.